docs/TodoList.md

```diff
@@ -585,10 +585,3 @@ We'll need another public method, waitFor(). Note that it returns a Promise that
 ```
 
 Now within the TodoStore callback we can explicitly wait for any dependencies to first update before moving forward.  However, if Store A waits for Store B, and B waits for A, then a circular dependency will occur.  A more robust dispatcher is required to flag this scenario with warnings in the console.
-
-The Future of Flux
```

Comment:
I would prefer leaving this mostly intact, perhaps a better way to update this could be:

```
A lot of people ask if Facebook will release Flux as an open source framework. Really, Flux is just an architecture, not a framework, but we have released `flux/utils` which is a small set of utilities we use at Facebook to build Flux applications. There are also many other great frameworks out there that enable the Flux architecture.
```

Also, don't remove the thanks at the end please :)